### PR TITLE
Fixed "Event Store Cloud" name in the cloud docs (e.g. usages of "EventStoreDB Cloud")

### PR DESCRIPTION
**Pull request by oskardudycz**
See also: https://github.com/EventStore/esc/pull/60.

### CODE REVIEW

File: docs/cloud/automation/terraform.md [hunk 1]
```diff
@@ -90,9 +90,9 @@ In the cloud console:
 
 ### Obtaining the `organization_id`
 
-You can use the [cloud console][cloud console organizations] or the [EvenStoreDB Cloud CLI][esc cli github releases] to find your organization id.
+You can use the [cloud console][cloud console organizations] or the [EventStoreDB Cloud CLI][esc cli github releases] to find your organization id.
 
-The command to obtain the organization from the [EvenStoreDB Cloud CLI][esc cli github releases] is:
+The command to obtain the organization from the [EventStoreDB Cloud CLI][esc cli github releases] is:
```
Comment (alexeyzimarev) [inline docs/cloud/automation/terraform.md]: I think this one is still wrong, as it's `esc-cli`, which is Event Store Cloud CLI.

**Review by alexeyzimarev — CHANGES_REQUESTED** [review]

File: docs/cloud/automation/terraform.md (hunk 1 above)
Comment (oskardudycz) [reply docs/cloud/automation/terraform.md]: I'm okay with updating that, but we'll need to also clarify the naming in the CLI repo https://github.com/EventStore/esc#evenstoredb-cloud-cli. It has `EvenStoreDB Cloud CLI`. Thoughts?

Comment (ylorph) [reply docs/cloud/automation/terraform.md]: we need to take that to @sigil66 
I'd honestly leave it as is , as it's the name in the readme  .

Comment (oskardudycz) [reply docs/cloud/automation/terraform.md]: @alexeyzimarev @ylorph 
I've sent PR also to the CLI repo: https://github.com/EventStore/esc/pull/60.

Updated usage in documentation in https://github.com/EventStore/documentation/pull/298/commits/c5b14b22ba8ecb27c8e332be814d3d3408c07e80.

**Review by ylorph — APPROVED** [review]

**Review by alexeyzimarev — APPROVED** [review]

**Pull request merged**